Yoshi/Yoshi/Yoshi.swift

```diff
@@ -69,7 +69,9 @@ public final class Yoshi {
             guard let percent: CGFloat = CGFloat(minimumForcePercent) else {
                 return false
             }
-            return touch.force >= touch.maximumPossibleForce * (percent / 100)
+            // Cannot use >= here since touch.force and touch.maximumPossibleForce equals 0
+            // on non-3D touch capable devices
+            return touch.force > touch.maximumPossibleForce * (percent / 100)
```

Comment:
Not to be difficult, but can't we just instead update the bool to check that maximumPossibleForce is > 0 ?

`return (touch.force > touch.maximumPossibleForce * (percent / 100)) && (touch.maximumPossibleForce > 0)`

Same effect, but then it also doesn't have the side effect of not handling touch.force == 6